serviceWrappers/linux/kLiveController

```diff
@@ -93,22 +97,18 @@ restart() {
     start
 }
 
-status() {
+###
+# check whether process is running based on /proc/[pid]/exe
+###
+my_status() {
     RETVAL=0
     for prefix in 0 1 ; do
         PIDFILE="$PID_DIR/${NAME}_${prefix}.pid"
         if [ -f $PIDFILE ] ; then
-	    VAL=$( ( cat ${PIDFILE} | xargs ps -p ) | wc -l )
-	    if [ $VAL -gt 1 ] ; then
-            echo "${NAME}_${prefix} is running with PID $( cat $PIDFILE )"
-            else
-            echo "${NAME}_${prefix} is not running but there is stale pidFile: $PIDFILE"
-            RETVAL=$(( $RETVAL + 1 ))
- 	    fi
-        else
-            echo "${NAME}_${prefix} is not running."
-	        RETVAL=$(( $RETVAL + 1 ))
-        fi
+            status -p ${PIDFILE}
+            if [ ! $? -eq 0 ]; then
```

Comment:
```
if [ ! $? -eq 0 ]; then
```
should be:
```
if [ $? -ne 0 ]; then
```
Also, as I wrote in https://kaltura.atlassian.net/browse/PLAT-7392
However, there's another problem here: you're checking the status of two different PIDs in the same function [in a for loop] but you do not return in the event status -p /var/run/kLiveController_0.pid /opt/kaltura/liveController/latest/lib/App.js fails, as a result, there may be a case where /var/run/kLiveController_0.pid does not contain a valid PID but /var/run/kLiveController_1.pid does and the status action will therefore return with RC 0, even though one of the two procs [the first one] is not running.
In the event /var/run/kLiveController_0.pid does not contain a valid PID, you should return with $?.